src/404.md

```diff
@@ -4,6 +4,6 @@ permalink: /404.html
 excludeFromSitemap: true
 layout: layouts/base.njk
 ---
-# 404: document not found
+## 404: document not found
```

Comment:
These should all remain `<h1>`s.

Reply:
It can either be ignored inline with `<!-- markdownlint-disable ... -->`, or could be removed if you have a separate layout that doesn't already have an H1 like the base.nkj does

Reply:
Either of those solutions sounds good to me.